docs/tutorials/best-practices/error-safety.md

```diff
@@ -329,7 +329,7 @@ You must move the `Safe` closer to the source of the error, as discussed before.
 ```Lua
 -- The `Safe` and the the `Computed` have swapped places.
 -- The error is now caught by the `Safe` instead of the `Computed`.
-local result = scope:Computed(function()
+local result = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
local result = scope:Computed(function(use)
```

Reply:
Use also shouldn't be provided as it is never used
```suggestion
local result = scope:Computed(function()
```

Reply:
reverted